### PR TITLE
Scale InfluxDB Clustered

### PR DESCRIPTION
**Pull request by sanderson**
Adds information about scaling InfluxDB Clustered.

- [x] Rebased/mergeable

### CODE REVIEW

**Review by jenniferplusplus — APPROVED** [review]
This all seems right, but I think it's also a good idea to discuss the ingester storage configuration. Unless that has its own page elsewhere?

If the ingesters have more storage available, then they can keep bigger WAL buffers, which helps with query performance and reduces pressure on the compactor. Also, obviously faster storage helps with query performance, in addition to just the quantity.

We expose storage configuration under `spec.package.spec.ingesterStorage`. That's another area where the options will depend on the underlying hardware in the cluster, and how it's configured.

File: content/influxdb/clustered/admin/scale-cluster.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+components depend on the technology and underlying provider used for each.
+{{% /note %}}
+
+Use the `spec.package.spec.resources` property in your `AppInstance` resource
+defined in your `influxdb.yml` to define system resource minimums and limits
+for each pod and the number of replicas per component.
+`requests` are the minimum that the Kubernetes scheduler should reserve for a pod.
+`limits` are the maximum that a pod should be allowed to use.
+
+Your `AppInstance` resource can included the following properties to define
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/scale-cluster.md:86]:
```suggestion
Your `AppInstance` resource can include the following properties to define
```

**Review by jstirnaman — APPROVED** [review]
A small fix, otherwise looks good pending other questions.

**Comment by sanderson** [comment]
Thanks @jenniferplusplus. I added information about the Ingester storage. To you mind fact checking me before I push this out?

File: content/influxdb/clustered/reference/glossary.md [hunk 2]
```diff
@@ -1131,15 +1131,17 @@ A statement that sets or updates the value stored in a variable.
...
+### WAL (Write-Ahead Log)
 
 The temporary cache for recently written points.
 To reduce the frequency that permanent storage files are accessed, InfluxDB
 caches new points in the WAL until their total size or age triggers a flush to
-more permanent storage. This allows for efficient batching of the writes into the TSM.
+more permanent storage. This allows for efficient batching of the writes into
+the storage engine.
 
 Points in the WAL can be queried and persist through a system reboot.
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/glossary.md:1142]:
```suggestion
Points in the WAL are queryable and persist through a system reboot.
```

**Review by jstirnaman — APPROVED** [review]
🔥

**Comment by dsprogis** [comment]
Hi Scott, great job!  Two recommendations:
1. There are two places where you describe the process of applying the changes, one is highlighted in green (which, to me signals action) and the other is not.  I like the green signal and suggest it should be applied to both.
2. I think the user would benefit from knowing what to expect after applying the change - something such as, "Once the file is saved, the changes will be applied immediately.  You will see <this> happen.  If scaling up, you will begin to see <this>.  If scaling down, you will begin to see <that>.  If you do not see the changes, consult <this> documentation for troubleshooting tips."

**Pull request merged**